src/System.Reflection.Metadata/src/System/Reflection/Metadata/MetadataReader.netstandard2.0.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Reflection.Metadata
+{
+    public sealed partial class MetadataReader
+    {
+        internal AssemblyName GetAssemblyName(StringHandle nameHandle, Version version, StringHandle cultureHandle, BlobHandle publicKeyHandle, AssemblyHashAlgorithm assemblyHashAlgorithm, AssemblyFlags flags)
+        {
+            string name = (!nameHandle.IsNil) ? GetString(nameHandle) : null;
```

Comment:
Should we handle this and throw an exception? Maybe an `InvalidDataException` or something; because passing null to `AssemblyName..ctor(string)` will throw a `NullReferenceException`

cc: @nguerrera